### PR TITLE
new question about now() function for SQL quiz

### PR DESCRIPTION
**Pull request by raykotab**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #XXXXX

### CODE REVIEW

File: src/data/sql-quiz.ts [hunk 1]
```diff
@@ -333,6 +333,17 @@ const sqlQuiz = [
...
   },
+  {
+    Question:
+      "In SQL, what is the command used to give a record a date and time value, from the system on execution?",
+    Answer: "NOW()",
+    Distractor1: "TIMESTAMP()",
+    Distractor2: "Y-M-D()",
+    Distractor3: "DATETIME()",
+    Explanation:
+      "The NOW() function gives the selected record a value of the current tiemstamp",
```
Comment (jdwilkin4) [inline src/data/sql-quiz.ts:344]: spelling error here

```suggestion
      "The NOW() function gives the selected record a value of the current timestamp",
```

File: src/data/sql-quiz.ts [hunk 2]
```diff
@@ -333,6 +333,17 @@ const sqlQuiz = [
     Explanation: "The DROP TABLE command is used to delete an entire table.",
     Link: "https://www.freecodecamp.org/news/learn-sql-queries-database-query-tutorial-for-beginners/",
   },
+  {
+    Question:
+      "In SQL, what is the command used to give a record a date and time value, from the system on execution?",
```
Comment (jdwilkin4) [inline src/data/sql-quiz.ts:338]: I don't think we need this comma here

```suggestion
      "In SQL, what is the command used to give a record a date and time value from the system on execution?",
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

File: src/data/sql-quiz.ts (hunk 1 above)
Comment (raykotab) [reply src/data/sql-quiz.ts]: Sorry for the silly error, and thanks for your work @jdwilkin4

**Review by jdwilkin4 — APPROVED** [review]
Hi @raykotab ! 

Congrats on your first contribution to our repository. 🎉
We look forward to future contributions. 👍

**Comment by raykotab** [comment]
thaaanks @jdwilkin4, really happy about it!

**Pull request merged**